content/en/docs/pipelines/installation/overview.md

```diff
@@ -169,6 +180,13 @@ Upgrade Support (**Alpha**)
 
 For upgrade via reinstallation (with existing data) instructions, refer to [Upgrading AI Platform Pipelines](/docs/gke/pipelines/upgrade/#ai-platform-pipelines).
 
+Google Cloud Integrations
+:
+  * You can deploy AI Platform Pipelines on [Google Cloud Console UI](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines).
```

Comment:
Minor suggestions (same as above):
- Google Cloud Console -> Cloud Console (similar to Cloud Shell), since this is a GCP-related guide (ref: https://cloud.google.com/cloud-console)

```diff
-  * You can deploy AI Platform Pipelines on [Google Cloud Console UI](...)
+  * You can deploy AI Platform Pipelines on [Cloud Console UI](...)
```